modules/calendar/db/migrate/20230607101213_populate_share_calendar_role_permission.rb

```diff
@@ -0,0 +1,20 @@
+class PopulateShareCalendarRolePermission < ActiveRecord::Migration[7.0]
+  def up
+    # From https://community.openproject.org/projects/openproject/work_packages/15339/activity
+    # -> For the migration path, all the roles that has the setting "View calendars" active will
+    # also by default have "Subscribe to iCalendars" active
+    role_ids_with_view_calendar_permission = RolePermission.where(permission: "view_calendar").distinct(:role_id).pluck(:role_id)
+
+    new_permissions = role_ids_with_view_calendar_permission.map do |role_id|
+      { role_id:, permission: "share_calendars" }
+    end
+
+    RolePermission.insert_all(new_permissions)
```

Comment:
We have a [nifty little helper](https://github.com/opf/openproject/blob/dev/db/migrate/migration_utils/permission_adder.rb) for this used e.g. in [this migration](https://github.com/opf/openproject/blob/dev/db/migrate/20190527095959_set_manage_board_permission.rb)